google/cloud/compute/v1/BUILD.bazel

```diff
@@ -149,14 +210,52 @@ java_gapic_assembly_gradle_pkg(
     ],
 )
 
+# Java Compute Small (for testing and prototyping purposes)
+java_proto_library(
+    name = "compute_small_java_proto",
+    deps = [":compute_small_proto"],
+)
+
+java_gapic_library(
+    name = "compute_small_java_gapic",
+    src = ":compute_small_proto_with_info",
+    gapic_yaml = "compute_gapic.yaml",
+    package = "google.cloud.compute.v1",
+    service_yaml = "compute_small_v1.yaml",
+    transport = "rest",
+    test_deps = [],
+    deps = [
+        ":compute_small_java_proto",
+    ],
+)
+
+#java_gapic_test(
```

Comment:
cruft?

Reply:
No, it is intentional. As of now java_gapic_test macro is written in a way that there can be only one instance of that in one build file. Need to fix the macro itself first (it is easy).